Content.Server/Forensics/Systems/ForensicScannerSystem.cs

```diff
@@ -33,6 +34,8 @@ public sealed class ForensicScannerSystem : EntitySystem
         [Dependency] private readonly ForensicsSystem _forensicsSystem = default!;
         [Dependency] private readonly TagSystem _tag = default!;
 
+        private static readonly ProtoId<TagPrototype>DNASolutionScannableTag = "DNASolutionScannable";
```

Comment:
```suggestion
        private static readonly ProtoId<TagPrototype> DNASolutionScannableTag = "DNASolutionScannable";
```
TIL this isn't a syntax error.

Reply:
huh, crazy

Reply:
Good catch!